c/src/detectors/c-random-fd-exhaustion/c-random-fd-exhaustion_non-compliant.c

```diff
@@ -0,0 +1,20 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+// {fact rule=c-random-fd-exhaustion@v1.0 defects=1}
+#include <fcntl.h>
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+
+int noncompliant() {
+    //NonCompliant : Failure to limit and close open file descriptors allows uncontrolled resource consumption, potentially impacting system stability and performance.
```

Comment:
1. Please shorten the comment to a single line
2. Comment seems to be misplaced